batch_runner.py

```
@@ -1,20 +1,174 @@
 import argparse
-from benchmark_runner import run_benchmark
+import json
+import time
+from pathlib import Path
+from benchmark_runner import run_benchmark, SCENARIO_CONFIGS, MODEL_CONFIGS
 
-def run_multiple_benchmarks(scenarios, models):
-    """Run multiple benchmark scenarios with multiple models"""
+def run_batch_benchmarks(scenarios, models, task_params=None):
+    """Run benchmarks for multiple scenarios and models."""
+    print(f"🚀 Starting batch benchmark run")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)